yarn.lock

```diff
@@ -1755,7 +1755,7 @@
   dependencies:
     prop-types "^15"
 
-"@dhis2/ui-core@^4.16.0", "@dhis2/ui-core@^4.6.1", "@dhis2/ui-core@^4.9.1":
```

Comment:
This should be kept to prevent previous bugs we've had (yarn insists on breaking these up again but all minor versions should resolve to `"4.16.0"`

Reply:
Fixed.